### ISSUE TITLE
load pre-trained vgg model problem

### ISSUE THREAD

**Issue by FrancisGee** [MaybeShewill-CV/lanenet-lane-detection#105]
你好，我用加载的vgg模型开始训练，训练参数和你的一致，为什么准确率不升反降
vgg16模型来自于[vgg16](https://github.com/machrisaa/tensorflow-vgg)

![2018-11-28 16-56-18](https://user-images.githubusercontent.com/18606896/49140196-9c3ba580-f32e-11e8-94d3-6c8d8e66d9c7.png)

**Comment by MaybeShewill-CV** [comment]
@FrancisGee 只有十几个epoch看不出来问题 训练多个epoch再看看:)

**Comment by cardwing** [comment]
@FrancisGee , it is common that the accuracy decreases in the first several episodes. The accuracy will increase as the training process proceeds.
